dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ResourcesController.java

```diff
@@ -124,11 +122,10 @@ public class ResourcesController extends BaseController {
     public Result<Object> createDirectory(@Parameter(hidden = true) @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
                                           @RequestParam(value = "type") ResourceType type,
                                           @RequestParam(value = "name") String alias,
-                                          @RequestParam(value = "description", required = false) String description,
                                           @RequestParam(value = "pid") int pid,
                                           @RequestParam(value = "currentDir") String currentDir) {
         // todo verify the directory name
-        return resourceService.createDirectory(loginUser, alias, description, type, pid, currentDir);
```

Comment:
could you add some desc to `incompatible.md` due to we change the public interface?